README.md

```diff
@@ -733,6 +730,149 @@ Error Connecting to auth service ...
 }
 ```
 
+## Parse import specifications
+
+This endpoint parses one or more import specification files in the staging area into a data
+structure (close to) ready for insertion into the Narrative bulk import cell.
+
+It can parse `.tsv`, `.csv`, and Excel (`.xls` and `.xlsx`) files. Templates for the currently
+supported data types are available in the `import_specifications/templates` directory of this repo.
```

Comment:
can add a link here

Reply:
Good idea, added links in a couple other places as well